debian/kubelet/debian/control

```diff
@@ -0,0 +1,16 @@
+Source: kubelet
+Section: misc
+Priority: optional
+Maintainer: Mike Danese <mikedanese@google.com>
+Build-Depends: debhelper (>= 8.0.0)
+Standards-Version: 3.9.4
+Homepage: https://kubernetes.io
+Vcs-Git: https://github.com/kubernetes/kubernetes.git
+Vcs-Browser: https://github.com/kubernetes/kubernetes
+
+Package: kubelet
+Architecture: amd64
+Depends: docker-engine (= 1.11.2-0~xenial), iptables (>= 1.4.21), k8s-cni (= 0.0.1-0),
```

Comment:
I think it's socat, iproute, util-linux (for nsenter & findmnt) and ethtool, isn't it?